notifier.go

```diff
@@ -75,6 +87,9 @@ func NewNotifier(projectId int64, projectKey string) *Notifier {
 	if wd, err := os.Getwd(); err == nil {
 		n.context["rootDirectory"] = wd
 	}
+	for i := 0; i < 10; i++ {
+		go n.worker()
```

Comment:
Am I correct in seeing that this configures the notifier with a channel to buffer up to 1000 notices, with 10 workers processing (POSTing) them? What happens when the channel is full and we get more notices?